apps/README.md

```diff
@@ -21,20 +22,37 @@ This `apps` directory contains the source-code and/or content for a number of HA
 
 ## HASH
 
-[HASH], is a block-based, data-centric, all-in-one workspace.
+[HASH] is a block-based, data-centric, AI-enabled "super app".
 
-See the [`README`](hash/README.md) in the [`hash`](hash) folder for a holistic description of the workspace, and instructions for running it or testing it.
+The [README in the `hash` directory](hash/README.md) provides a more detailed overview of the app, and instructions for running it.
 
-The workspace depends on a suite of constituent services, which are briefly described below, keep in mind that they are not guaranteed to be useful when ran independently.
+The application depends on a suite of constituent services, which are briefly described below. Please note that these are not designed or guaranteed to be useful when ran independently.
```

Comment:
Do we want to mention, that we don't guarantee to avoid breaking changes?

Reply:
I think we should do that on a per-app level.